src/promptflow-azure/promptflow/azure/_storage/cosmosdb/summary.py

```diff
@@ -126,11 +128,11 @@ def _persist_running_item(self, client: ContainerProxy):
         safe_create_cosmosdb_item(client, item)
 
     def _persist_line_run(self, client: ContainerProxy):
-        attributes: dict = self.span._content[SpanFieldName.ATTRIBUTES]
+        attributes: dict = self.span.attributes
 
-        session_id = self.span.session_id
-        start_time = self.span._content[SpanFieldName.START_TIME]
-        end_time = self.span._content[SpanFieldName.END_TIME]
+        session_id = self.session_id
+        start_time = self.span.start_time.isoformat()
+        end_time = self.span.end_time.isoformat()
 
         # Span's original format don't include latency, so we need to calculate it.
         # Convert ISO 8601 formatted strings to datetime objects
```

Comment:
Here already do the start_time.isoformat(). Do we still need to call the "start_time.replace("Z", "+00:00")" to get total_seconds? Maybe we can use span.start_time and span.end_time to get the duration?

Reply:
cc @huaiyan